### ISSUE TITLE
[DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead

### ISSUE THREAD

**Issue by Aeonrush** [pwnall/node-xhr2#33]
Hi
I worked with Angular Universal and I got a DeprecationWarning. While investigation I found that the warning happened in xhr2
To reproduce I made
```
git clone git://github.com/pwnall/node-xhr2.git
cd node-xhr2
npm install
npm pack
npm test
```
And I got a DeprecationWarning
```
XMLHttpRequest
    level 2 events
      for a successful fetch with Content-Length set
(node:10531) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
```
I found an article [Porting to the ```Buffer.from()```/```Buffer.alloc()``` API](https://nodejs.org/en/docs/guides/buffer-constructor-deprecation/)

```
node -v
v10.15.3
```

I can make a PR
Thank you
P.S. Sorry for my English

**Comment by Avcajaraville** [comment]
Having same deprecation warning on my app due to angular dependency on this module

**Comment by mtsaryk** [comment]
+1

**Comment by pwnall** [comment]
Fixed in release 0.2.0.
